tests/models/test_fluid_mass_balance.py

```diff
@@ -415,10 +415,24 @@ def test_unit_conversion(units):
     class Model(SquareDomainOrthogonalFractures, SinglePhaseFlow):
         """Single phase flow model in a domain with two intersecting fractures."""
 
+        def bc_values_pressure(self, boundary_grid: pp.BoundaryGrid) -> np.ndarray:
+            """Ensure nontrivial solution."""
+            vals = self.fluid.pressure() * np.ones(boundary_grid.num_cells)
+            faces = self.domain_boundary_sides(boundary_grid).east
+            vals[faces] += self.fluid.convert_units(1e10, "Pa")
```

Comment:
Isn't this a somewhat extreme value?

Reply:
Heh, yeah it is.. I was playing around with values to check the solution. Forgot to change back to something more meaningful (e.g. 1e5 I assume). Should it be changed back?

Reply:
Yes, please.